ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -71,7 +71,7 @@ def get_closest_waypoint_idx(self):
         val = np.dot(cl_vect-prev_vect, pos_vect-cl_vect)
 
         if val > 0:
-            closest_idx = closest_idx + 1 % len(self.waypoints_2d)
+            closest_idx = (closest_idx + 1) % len(self.waypoints_2d)
```

Comment:
This could be the reason the car gets lost in the end of the lap. Initial attempt to fix it